src/repos/FlyerRepo.ts

```diff
@@ -70,6 +70,18 @@ const getFlyersByCategorySlug = async (
     });
 };
 
+/**
+ * Looks for unique category slugs for all the Flyers
+ * @returns a list of Strings with all the category names. 
+ */
+const getAllFlyerCategories = async (
```

Comment:
This function actually doesn't need to be `async`, since all you're doing here is returning a promise, which should actually be a synchronous action. Async/await stuff can be kind of confusing -- as a general rule of thumb, only mark a function `async` if you're using the keyword `await` inside of it!